spi/storage/storage.go

```diff
@@ -33,13 +33,40 @@ type StoreConfiguration struct {
 	TagNames []string `json:"tagNames,omitempty"`
 }
 
+// SortOrder specifies the sort order of query results.
+type SortOrder int
+
+const (
+	// SortAscending indicates that the query results must be sorted in ascending order.
+	SortAscending SortOrder = iota
+	// SortDescending indicates that the query results must be sorted in descending order.
+	SortDescending
+)
+
+// SortOptions sets the order that results from an Iterator will be returned in. Sorting is based on the tag values
```

Comment:
Let me know if you are able to understand this - I had some trouble trying to find clear, concise wording to describe how the sorting works.

(and if you're having trouble understanding this - definitely let me know!)